### ISSUE TITLE
Editor is down

### ISSUE THREAD

**Issue by runemadsen** [processing/p5.js-web-editor#686]
![screen shot 2018-09-09 at 4 31 00 pm](https://user-images.githubusercontent.com/192021/45262660-c24b5b00-b44d-11e8-97e0-58b62c8c811e.png)

**Comment by slowizzm** [comment]
Same here, started around 9-10p CT ... and still having issues this morning.

**Comment by Lordgoderic** [comment]
The guest account will load but I get the 502 error when trying to login via my google account, which was working yesterday.

**Comment by shiffman** [comment]
Thanks all, we're aware of the issue and @catarak is looking into it! Stay tuned!

**Comment by catarak** [comment]
Just fixed it, so sorry about this! 

For documentation purposes, what happened was that a Kubernetes cronjob ran to update all of the p5.js examples (https://editor.p5js.org/p5/sketches), but it didn't run correctly and corrupted the database. Yay! I removed the cronjob so that won't happen again.

This is also, just, hilarious, since I decided to use Kubernetes to prevent stuff like this from happening, and tried rolling back the image to a version I knew worked, and then upon seeing the same error, I realized it had to do with the database and not the app itself.
